src/west/commands/project.py

```diff
@@ -744,16 +739,15 @@ def _west_project():
 
 
 def _update_west(rebase, keep_descendants):
+    _banner('self-updating west:')
     with _error_context(_FAILED_UPDATE_MSG):
         project = _west_project()
-        log.dbg(project.format('Updating {name_and_path}'))
-
         old_sha = _sha(project, 'HEAD')
         _update(project, rebase, keep_descendants)
 
         if old_sha != _sha(project, 'HEAD'):
-            log.inf(project.format(
-                'Updated {name_and_path} to {revision} (from {url}).'))
+            _msg(project.format(
```

Comment:
We could change `_msg()` signature to `_msg(project, fmt, *args, **kwargs)` and prepend `'{name}: '` to `fmt` if we wanted.

Then calls would look like `_msg(project, 'updated to {revision} (from {url})')`.

I didn't think it was worth it though, but am open to the change.

Reply:
I like the  proposed change in terms of calls to it would look like, but we can do this in a follow-up PR.

Reply:
Sure, that's fine.